tasks/main.yml

```diff
@@ -16,10 +16,17 @@
     - base_enable_elts is defined
     - base_enable_elts == true
     - ansible_distribution_release == "wheezy" or ansible_distribution_release == "jessie" or ansible_distribution_release == "stretch" or ansible_distribution_release == "buster"
+
+- name: Make sure /etc/apt/debian.sources is absent
+  file:
+    path: /etc/apt/debian.sources
+    state: absent
+
 - name: Provide /etc/apt/sources.list
   template: src=templates/apt/sources.list.j2 dest=/etc/apt/sources.list owner=root group=root mode=0644
   notify: apt-get update
   when: ansible_distribution == 'Debian'
+
```

Comment:
nitpick: hm, by intention?

and while at it, adjust commit message?

I suppose the:

```
This role currently does support and expect anything else then the classic one-line-style format.```

was meant to be something like:

``This role currently does support only the classic one-line-style format.```

or so?